po/zh_CN.po

```diff
@@ -133,22 +133,22 @@ msgstr "生成压缩归档......"
 #: ../sos/sosreport.py:1498
 #, python-format
 msgid " %s while finalizing archive"
-msgstr ""
+msgstr "%s存档完成"
 
 #: ../sos/sosreport.py:1517
 #, python-format
 msgid "Error moving directory: %s"
-msgstr ""
+msgstr "从%s中移出是出错"
 
 #: ../sos/sosreport.py:1540
 #, python-format
 msgid "Error moving archive file: %s"
-msgstr ""
+msgstr "移除%s时出错"
 
 #: ../sos/sosreport.py:1558
 #, python-format
 msgid "Error moving checksum file: %s"
-msgstr ""
+msgstr "移除%s时出错"
```

Comment:
It's the same string like the previous one? Why dont differentiate "archive" and "checksum"? (but it's a minor nitpick, one gets that info from the context / from the reported filename)